src/app/login/page.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import { FC, useState } from 'react';
+import Link from 'next/link';
+
+const LoginForm: FC = () => {
+  const [userData, setUserData] = useState({
+    email: '',
+    password: '',
+  });
+
+  return (
+    <section className="sec-cont login-container">
+      <div>
+        <h1>Inloggning</h1>
+        <form action="">
+          <label htmlFor="email">Email</label>
+          <input
+            onChange={(e) => {
+              e.preventDefault();
```

Comment:
The prevent default doesn't go here, it has to be in the form using the onSubmit event.

**Tips**
This is another way to update the useState with less code:
```tsx
const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
    setUserData({
      ...userData,
      [e.target.id]: e.currentTarget.value,
    });
};
  
<input
    onChange={handleChange}
    className="form-input-field"
    type="text"
    id="email"
/>

<input
    onChange={handleChange}
    className="form-input-field"
    type="password"
    id="password"
/>
```

Reply:
I took your shorter syntax for setUserData but I like to have the event inside jsx so typescript can use type inference:
![image](https://user-images.githubusercontent.com/112066906/234966245-8ed9bd29-8bab-4d09-8c94-03bec7905932.png)